addon/index.js

```diff
@@ -0,0 +1,41 @@
+import Rollbar from "rollbar";
+import Ember from 'ember';
```

Comment:
```suggestion
import Rollbar from "rollbar";
import Ember from 'ember';

export * as Rollbar from "rollbar";
```

Or.. I'd just avoid reexporting rollbar and let people import it themselves?

Reply:
Rollbar won't be a direct dependency of the app. But maybe it should be then? 🤔  Maybe this addon should just bring the integration layer nothing else?